lua/lspconfig/server_configurations/julials.lua

```diff
@@ -1,5 +1,54 @@
 local util = require 'lspconfig.util'
 
+local function activate_env(path)
+  local bufnr = vim.api.nvim_get_current_buf()
+  local julials_clients = util.get_lsp_clients { bufnr = bufnr, name = 'julials' }
```

Comment:
`vim.lsp.get_clients()`

Reply:
Ah, I misunderstood the previous discussion. I thought it should still use the `get_clients` utility since the rest of the codebase supports neovim 0.9 and then the 0.9 support would be dropped in a later PR

Reply:
resolved in latest commit